src/python/pants/backend/python/register.py

```diff
@@ -50,6 +51,7 @@ def build_file_aliases():
 
 
 def register_goals():
+  task(name='partition', action=PartitionTargets).install('pyprep')
```

Comment:
I don't think this is specific to python. Can probably either put it in the `'bootstrap'` goal instead, or remove it into its own top-level goal (by calling `install()` with no arguments). If you do the later thing, would recommend giving it a longer name in order to not squat a name that an explicitly called goal might use `compute-partitions` for example.